### DISCUSSION TITLE
Option to Save file with generated id and title of note ID+Title

### DISCUSSION THREAD

**Discussion by mardatasci** [Zettlr/Zettlr#2802] [Q&A]
Hi,

Is there a way for when a new note is created that the note title is saved with the id?  Also, is there an option to have that also as the title

ID+Title for both filename and inside the note much like The Archive or Sublime_zk and obsidian?

I can't seem to find the information if it exists

**Comment by nathanlesage** [comment]
If you save a file, you need to manually add any title to the file, Zettlr can't do it automatically.
